src/content/guides/getting-started.md

```diff
@@ -20,15 +20,18 @@ Webpack is used to compile JavaScript modules. Once [installed](/guides/installa
 
 ## Basic Setup
 
-First let's create a directory, initialize npm, and [install webpack locally](/guides/installation#local-installation):
+First let's create a directory, initialize npm, [install webpack locally](/guides/installation#local-installation), and install the webpack-cli (which we'll need to run webpack from the command line (CLI)):
```

Comment:
Can we rephrase the sentence inside the parenthesis like _(the tool for running webpack on the command line)_? I feel it lighter and less over explanatory.

Reply:
Maybe we should wait for https://github.com/webpack/webpack/pull/6467? In that case it would be installed along webpack

Reply:
I won't honestly, we can't block documentation for a PR that has conflicts and uncertainty around when it could be merged.

If that changes in the future then we can quickly come and change it.

Reply:
Yeah, it seems stalled :/